.github/workflows/golangci-lint.yml

```diff
@@ -16,7 +16,7 @@ jobs:
         with:
           go-version: 1.18
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3.3.1
+        uses: golangci/golangci-lint-action@v3.4.0
```

Comment:
You are somehow picking up changes not in your PR.  Make sure you are submitting what you intend.

Reply:
i think this is from a dependabot that's already merged into master, but not part of this v1.8.1-staging branch so gets pulled in automatically. @martinstibbe do you know if there any easy way to remove of these future PRs? or if not, any concerns if we leave as is?

Reply:
I have rebased 1.8.1 staging with master so lint version should match up.

Reply:
Thanks!